src/OpenTracing/SpanOptions.php

```diff
@@ -18,12 +18,12 @@
     private $references = [];
 
     /**
-     * @var Tag[]
+     * @var array
      */
     private $tags = [];
 
     /**
-     * @var mixed
+     * @var int|float|\DateTime
      */
     private $startTime;
 
```

Comment:
This provides more flexibility about it, you can either pass the context or the span, in terms of tags it validates that the keys are strings (as PHP has this mixed concept of array = slice + map) and validates that either you pass the `child_of` or the references. Makes sense?